openedx/core/djangoapps/user_authn/views/login_form.py

```diff
@@ -89,15 +89,15 @@ def get_login_session_form(request):
 
     # Translators: This label appears above a field on the login form
     # meant to hold the user's email address.
-    email_label = _(u"Email")
+    email_label = _("Email")
 
     # Translators: This example email address is used as a placeholder in
     # a field on the login form meant to hold the user's email address.
-    email_placeholder = _(u"username@domain.com")
+    email_placeholder = _("example@domain.com")
```

Comment:
I wonder if we should also update the comment to note that we want to avoid this misunderstanding in the future. (Do translators see the diff?)